receiptui/src/main/java/com/hyperwallet/android/ui/receipt/viewmodel/TabbedListReceiptsViewModel.java

```diff
@@ -48,6 +48,7 @@
     private MutableLiveData<Boolean> retryListReceipts = new MutableLiveData<Boolean>();
     public ProgramModel mProgramModel;
     private boolean mIsInitialized;
+    private boolean mIsPortraitMode;
```

Comment:
Why is this variable introduced? should not it be always portrait?

Reply:
UI-sdk, common sdk, it has the option to set portrait/landscape Hyperwallet app to set as value as true to lock portrait only.